prov/rxm/src/rxm_attr.c

```diff
@@ -74,6 +77,7 @@ struct fi_domain_attr rxm_domain_attr = {
 
 struct fi_fabric_attr rxm_fabric_attr = {
 	.prov_version = FI_VERSION(RXM_MAJOR_VERSION, RXM_MINOR_VERSION),
+	.prov_name = rxm_prov_name
```

Comment:
We don't set the prov_name anymore. It would be handled by the framework. See #2808